app/code/core/Mage/Customer/Model/Resource/Customer/Collection.php

```diff
@@ -35,6 +35,13 @@
 class Mage_Customer_Model_Resource_Customer_Collection extends Mage_Eav_Model_Entity_Collection_Abstract
 {
     /**
+     * Model event prefix
+     *
+     * @var string
+     */
+    protected $_eventPrefix = 'eav_customer_customer_collection';
```

Comment:
I don't think the `eav_` makes a lot of sense, does it? I understand why you added, so you didn't need to use `resource` in the prefix. I think that using `resource` is more logical here. Unless you are basing the even prefix on an existing example in the core code that you can show me.